### PR TITLE
New migration: fenrir_id column added.

### PR DESCRIPTION
**Pull request by lrocheWB**
# Introduction

- Fix PSR errors
- Migration 003: 
    - "fenrir_id" column added in "tr_customer_cus" table.
    - Create NMM users in FenrirAPI
    - Then, update fenrir_id

## Dependencies
Depends of:
- [x] https://github.com/CanalTP/MigrationBundle/pull/1
- [ ] https://github.com/CanalTP/NMM/pull/182

## Testing
```
php app/console claroline:migration:upgrade CanalTPNmmPortalBundle
php app/console claroline:migration:downgrade CanalTPNmmPortalBundle
```

### CODE REVIEW

**Comment by Netmisa** [comment]
Jenckins could you test this please.

**Comment by Netmisa** [comment]
Jenkins could you test this please.

**Comment by Netmisa** [comment]
Jenkins could you test this please.

**Comment by Netmisa** [comment]
You need to add "canaltp/fenrir-api-client-bundle" in composer.json. If you doesn't have this, you can do request to FenrirApi.

File: Migrations/pdo_pgsql/Version003.php [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    {
+        return self::VERSION;
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function preUp(Schema $schema)
+    {
+        $this->tyrClient = new Client($this->container->getParameter('nmm.tyr.url'));
```
Comment (Netmisa) [inline Migrations/pdo_pgsql/Version003.php]: You need to use this "canaltp/fenrir-api-client-bundle". It's a bundle to request FenrirApi.

File: Migrations/pdo_pgsql/Version003.php [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        /* Create NMM origin with FenrirAPI */
+        $this->createNmmOrigin();
+
+        /* Update fenrir_id field in tr_customer_cus (NMM) */
+        $this->createFenrirUsersAndUpdateFenrirId();
+    }
+
+    private function createNmmOrigin()
+    {
+        $request = $this->fenrirClient->post("origins", [], ["name" => self::NMM_ORIGIN]);
```
Comment (datanel) [inline Migrations/pdo_pgsql/Version003.php]: it means a contributor needs to install fenrir? I think of Tisseo

Comment (Netmisa) [inline Migrations/pdo_pgsql/Version003.php]: Yes it's adding here : https://github.com/CanalTP/NmmPortalBundle/pull/17/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R41 in depencies.

**Pull request closed without merging**